src/main/java/com/softserveinc/dokazovi/controller/UserController.java

```diff
@@ -108,16 +110,25 @@ public ResponseEntity<Page<UserDTO>> getAllExpertsByDirectionsAndByRegionsOrdere
 				.body(userService.findAllExperts(userSearchCriteria, pageable));
 	}
 
+	@GetMapping(USER_ALL_EXPERTS_FOR_ADMINISTRATION_PURPOSE)
+	@ApiPageable
+	@ApiOperation(value = "Get all experts for admin page + sorting")
+	@PreAuthorize("hasAuthority('EDIT_AUTHOR')")
+	public ResponseEntity<Page<UserDTO>> getAllExpertsFprAdminPage(
+			@PageableDefault(size = 25) Pageable pageable, UserSearchCriteria userSearchCriteria) {
+		return ResponseEntity
+				.status(HttpStatus.OK)
+				.body(userService.findAllExperts(userSearchCriteria, pageable));
```

Comment:
Please advise why you decided to add new endpoint which provides the same data as the `USER_ALL_EXPERTS` above?